{"jobid": "c180cbb6-aa4f-42a7-ac54-d6429b2a47a9", "email": "wjw00527@hotmail.com", "model": ["1", "4"], "upload_timestamp": 1725896565.958725, "upload_time": "2024-09-09T15:42:45Z", "error": ["/home/wujiamin/.local/lib/python3.10/site-packages/statsmodels/base/model.py:607: ConvergenceWarning: Maximum Likelihood optimization failed to converge. Check mle_retvals\n  warnings.warn(\"Maximum Likelihood optimization failed to \"\n/home/wujiamin/.local/lib/python3.10/site-packages/statsmodels/base/model.py:607: ConvergenceWarning: Maximum Likelihood optimization failed to converge. Check mle_retvals\n  warnings.warn(\"Maximum Likelihood optimization failed to \"\n/home/wujiamin/.local/lib/python3.10/site-packages/statsmodels/base/model.py:607: ConvergenceWarning: Maximum Likelihood optimization failed to converge. Check mle_retvals\n  warnings.warn(\"Maximum Likelihood optimization failed to \"\n/home/wujiamin/.local/lib/python3.10/site-packages/statsmodels/discrete/discrete_model.py:2383: RuntimeWarning: overflow encountered in exp\n  return 1/(1+np.exp(-X))\n/home/wujiamin/.local/lib/python3.10/site-packages/statsmodels/discrete/discrete_model.py:2441: RuntimeWarning: divide by zero encountered in log\n  return np.sum(np.log(self.cdf(q * linpred)))\n/home/wujiamin/.local/lib/python3.10/site-packages/statsmodels/base/model.py:607: ConvergenceWarning: Maximum Likelihood optimization failed to converge. Check mle_retvals\n  warnings.warn(\"Maximum Likelihood optimization failed to \"\n/home/wujiamin/.local/lib/python3.10/site-packages/statsmodels/discrete/discrete_model.py:2383: RuntimeWarning: overflow encountered in exp\n  return 1/(1+np.exp(-X))\n/home/wujiamin/.local/lib/python3.10/site-packages/statsmodels/discrete/discrete_model.py:2441: RuntimeWarning: divide by zero encountered in log\n  return np.sum(np.log(self.cdf(q * linpred)))\n/home/wujiamin/.local/lib/python3.10/site-packages/statsmodels/discrete/discrete_model.py:2383: RuntimeWarning: overflow encountered in exp\n  return 1/(1+np.exp(-X))\n/home/wujiamin/.local/lib/python3.10/site-packages/statsmodels/discrete/discrete_model.py:2441: RuntimeWarning: divide by zero encountered in log\n  return np.sum(np.log(self.cdf(q * linpred)))\n/home/wujiamin/.local/lib/python3.10/site-packages/statsmodels/base/model.py:607: ConvergenceWarning: Maximum Likelihood optimization failed to converge. Check mle_retvals\n  warnings.warn(\"Maximum Likelihood optimization failed to \"\n/home/wujiamin/.local/lib/python3.10/site-packages/statsmodels/discrete/discrete_model.py:2383: RuntimeWarning: overflow encountered in exp\n  return 1/(1+np.exp(-X))\n/home/wujiamin/.local/lib/python3.10/site-packages/statsmodels/discrete/discrete_model.py:2441: RuntimeWarning: divide by zero encountered in log\n  return np.sum(np.log(self.cdf(q * linpred)))\n/home/wujiamin/.local/lib/python3.10/site-packages/statsmodels/discrete/discrete_model.py:2383: RuntimeWarning: overflow encountered in exp\n  return 1/(1+np.exp(-X))\n/home/wujiamin/.local/lib/python3.10/site-packages/statsmodels/discrete/discrete_model.py:2441: RuntimeWarning: divide by zero encountered in log\n  return np.sum(np.log(self.cdf(q * linpred)))\n/home/wujiamin/.local/lib/python3.10/site-packages/statsmodels/discrete/discrete_model.py:2383: RuntimeWarning: overflow encountered in exp\n  return 1/(1+np.exp(-X))\n/home/wujiamin/.local/lib/python3.10/site-packages/statsmodels/discrete/discrete_model.py:2441: RuntimeWarning: divide by zero encountered in log\n  return np.sum(np.log(self.cdf(q * linpred)))\n/home/wujiamin/.local/lib/python3.10/site-packages/statsmodels/discrete/discrete_model.py:2383: RuntimeWarning: overflow encountered in exp\n  return 1/(1+np.exp(-X))\n/home/wujiamin/.local/lib/python3.10/site-packages/statsmodels/discrete/discrete_model.py:2441: RuntimeWarning: divide by zero encountered in log\n  return np.sum(np.log(self.cdf(q * linpred)))\n/home/wujiamin/.local/lib/python3.10/site-packages/statsmodels/base/model.py:607: ConvergenceWarning: Maximum Likelihood optimization failed to converge. Check mle_retvals\n  warnings.warn(\"Maximum Likelihood optimization failed to \"\n/home/wujiamin/.local/lib/python3.10/site-packages/statsmodels/base/model.py:607: ConvergenceWarning: Maximum Likelihood optimization failed to converge. Check mle_retvals\n  warnings.warn(\"Maximum Likelihood optimization failed to \"\n/home/wujiamin/.local/lib/python3.10/site-packages/statsmodels/discrete/discrete_model.py:2383: RuntimeWarning: overflow encountered in exp\n  return 1/(1+np.exp(-X))\n/home/wujiamin/.local/lib/python3.10/site-packages/statsmodels/discrete/discrete_model.py:2441: RuntimeWarning: divide by zero encountered in log\n  return np.sum(np.log(self.cdf(q * linpred)))\n/home/wujiamin/.local/lib/python3.10/site-packages/statsmodels/base/model.py:607: ConvergenceWarning: Maximum Likelihood optimization failed to converge. Check mle_retvals\n  warnings.warn(\"Maximum Likelihood optimization failed to \"\n/home/wujiamin/.local/lib/python3.10/site-packages/statsmodels/base/model.py:607: ConvergenceWarning: Maximum Likelihood optimization failed to converge. Check mle_retvals\n  warnings.warn(\"Maximum Likelihood optimization failed to \"\n/home/wujiamin/.local/lib/python3.10/site-packages/statsmodels/base/model.py:607: ConvergenceWarning: Maximum Likelihood optimization failed to converge. Check mle_retvals\n  warnings.warn(\"Maximum Likelihood optimization failed to \"\n/home/wujiamin/.local/lib/python3.10/site-packages/statsmodels/discrete/discrete_model.py:2383: RuntimeWarning: overflow encountered in exp\n  return 1/(1+np.exp(-X))\n/home/wujiamin/.local/lib/python3.10/site-packages/statsmodels/base/model.py:607: ConvergenceWarning: Maximum Likelihood optimization failed to converge. Check mle_retvals\n  warnings.warn(\"Maximum Likelihood optimization failed to \"\n/home/wujiamin/.local/lib/python3.10/site-packages/statsmodels/base/model.py:607: ConvergenceWarning: Maximum Likelihood optimization failed to converge. Check mle_retvals\n  warnings.warn(\"Maximum Likelihood optimization failed to \"\n/home/wujiamin/.local/lib/python3.10/site-packages/statsmodels/base/model.py:607: ConvergenceWarning: Maximum Likelihood optimization failed to converge. Check mle_retvals\n  warnings.warn(\"Maximum Likelihood optimization failed to \"\n/home/wujiamin/.local/lib/python3.10/site-packages/statsmodels/base/model.py:607: ConvergenceWarning: Maximum Likelihood optimization failed to converge. Check mle_retvals\n  warnings.warn(\"Maximum Likelihood optimization failed to \"\n/home/wujiamin/.local/lib/python3.10/site-packages/statsmodels/discrete/discrete_model.py:2383: RuntimeWarning: overflow encountered in exp\n  return 1/(1+np.exp(-X))\n/home/wujiamin/.local/lib/python3.10/site-packages/statsmodels/discrete/discrete_model.py:2441: RuntimeWarning: divide by zero encountered in log\n  return np.sum(np.log(self.cdf(q * linpred)))\n/home/wujiamin/.local/lib/python3.10/site-packages/statsmodels/base/model.py:607: ConvergenceWarning: Maximum Likelihood optimization failed to converge. Check mle_retvals\n  warnings.warn(\"Maximum Likelihood optimization failed to \"\nTraceback (most recent call last):\n  File \"/mnt/md0/Public/PCMA_Web_sever/PCMA/src/PCMA1.py\", line 23, in <module>\n    main(args.data_dir,args.fun)\n  File \"/mnt/md0/Public/PCMA_Web_sever/PCMA/src/PCMA1.py\", line 8, in main\n    pcma1(\n  File \"/mnt/md0/Public/PCMA_Web_sever/PCMA/src/pcma/pcma1.py\", line 202, in pcma1\n    sankey(Bacteria_dir, \n  File \"/mnt/md0/Public/PCMA_Web_sever/PCMA/src/pcma/sankey.py\", line 20, in sankey\n    data = pd.read_csv(output_data_dir)\n  File \"/home/wujiamin/.local/lib/python3.10/site-packages/pandas/util/_decorators.py\", line 311, in wrapper\n    return func(*args, **kwargs)\n  File \"/home/wujiamin/.local/lib/python3.10/site-packages/pandas/io/parsers/readers.py\", line 680, in read_csv\n    return _read(filepath_or_buffer, kwds)\n  File \"/home/wujiamin/.local/lib/python3.10/site-packages/pandas/io/parsers/readers.py\", line 575, in _read\n    parser = TextFileReader(filepath_or_buffer, **kwds)\n  File \"/home/wujiamin/.local/lib/python3.10/site-packages/pandas/io/parsers/readers.py\", line 933, in __init__\n    self._engine = self._make_engine(f, self.engine)\n  File \"/home/wujiamin/.local/lib/python3.10/site-packages/pandas/io/parsers/readers.py\", line 1217, in _make_engine\n    self.handles = get_handle(  # type: ignore[call-overload]\n  File \"/home/wujiamin/.local/lib/python3.10/site-packages/pandas/io/common.py\", line 789, in get_handle\n    handle = open(\nFileNotFoundError: [Errno 2] No such file or directory: 'var/PCMA/userdata/a8963cb76a5dfb370dc574c41ce55a1d6b6abbb0aa84f8e9039ad396f23b2c22a5a3fa560d4d4fd3988ded83a8038bf0944df406c5a8621193ade634254b1e74/pcma1/result/data/sankey_data_dir.csv'\n", ""]}